native_lib/DDGSyncCrypto.c

```diff
@@ -166,15 +166,18 @@ DDGSyncCryptoResult ddgSyncSeal(
     unsigned char *message,
     unsigned long long messageLength) {
 
-    unsigned char output[crypto_box_SEALBYTES + messageLength];
+    unsigned char* output = new unsigned char[crypto_box_SEALBYTES + messageLength];
```

Comment:
You're using `new` and `delete[]` here which are C++ keywords. Prefer to use `malloc` and `free` for C.